templates/rancher/overlay/opt/corral/rancher/install-cert-manager.sh

```diff
@@ -4,13 +4,32 @@ set -ex
 helm repo add jetstack https://charts.jetstack.io
 helm repo update
 
-helm upgrade \
-  --install \
-  --create-namespace \
-  -n cert-manager \
-  --set installCRDs=true \
-  --version v1.8.0 \
-  --wait \
-  cert-manager jetstack/cert-manager
+if [ -z "${CORRAL_registry_fqdn}" ]; then
```

Comment:
is there any plan to test using signed/letsencrypt certs as well?

Reply:
I'm not sure, maybe Izaac can answer this is code he put in.

Reply:
This is the first iteration. So the answer is yes but not in these initial package PRs